docs/design/copy-tracking.md

```diff
@@ -176,17 +833,29 @@ A well known and thorny problem in Mercurial occurs in the following scenario:
 1.  Create a new file A
 1.  Create new commits on top that make changes to file A
 1.  Whoops, I should rename file A to B. Do so, amend the first commit.
-1.  Because the first commit created file A, there is no rename to record; it's changing to a commit that instead creates file B.
+1.  Because the first commit created file A, there is no rename to record; it's
+    changing to a commit that instead creates file B.
 1.  All child commits get sad on evolve
 
 In jj, we have an opportunity to fix this because all rebasing occurs atomically
 and transactionally within memory. The exact implementation of this is yet to be
 determined, but conceptually the following should produce desirable results:
 
 1.  Rebase commit A from parents [B] to parents [C]
-1.  Get copy records from [D]->[B] and [D]->[C], where [D] are the common ancestors of [B] and [C]
-1.  DescendantRebaser maintains an in-memory map of commits to extra copy info, which it may inject into (2). When squashing a rename of a newly created file into the commit that creates that file, DescendentRebase will return this rename for all rebases of descendants of the newly modified commit. The rename lives ephemerally in memory and has no persistence after the rebase completes.
-1.  A to-be-determined algorithm diffs the copy records between [D]->[B] and [D]->[C] in order to make changes to the rebased commit. This results in edits to renamed files being propagated to those renamed files, and avoiding conflicts on the deletion of their sources. A copy/move may also be undone in this way; abandoning a commit which renames A->B should move all descendant edits of B back into A.
+1.  Get copy records from [D]->[B] and [D]->[C], where [D] are the common
+    ancestors of [B] and [C]
+1.  MutableRepo maintains an in-memory map of commits to extra copy info,
+    which it may inject into (2). When squashing a rename of a newly created
+    file into the commit that creates that file, MutableRepo will return
+    this rename for all rebases of descendants of the newly modified commit. The
+    rename lives ephemerally in memory and has no persistence after the rebase
+    completes.
+1.  A to-be-determined algorithm diffs the copy records between [D]->[B] and
+    [D]->[C] in order to make changes to the rebased commit. This results in
+    edits to renamed files being propagated to those renamed files, and avoiding
+    conflicts on the deletion of their sources. A copy/move may also be undone
+    in this way; abandoning a commit which renames A->B should move all
+    descendant edits of B back into A.
 
 ### Conflicts
```

Comment:
Is this section still relevant? It feels a bit out of date.

I'd either delete it or say explicitly that this section doesn't discuss the conflicts from copy-tracking + content changes. (For example, you could add a heading for that and mark it as TODO).